cmp/grid/GridModel.ts

```diff
@@ -353,6 +353,21 @@ export interface GridConfig {
      */
     enableFullWidthScroll?: boolean;
 
+    /**
+     * Set to true to disable scroll optimization for large grids, where we proactively update the
+     * row heights in ag-grid whenever the data changes to avoid hitching while quickly scrolling
+     * through large grids.
```

Comment:
I would be inclined to just put this quietly in "experimental" if we think its farily likely that we eventually just remove it.